server/index.js

```diff
@@ -1,22 +1,31 @@
-var express = require('express');
-var connection = require('../database/index.js');
-var path = require('path');
-var port = 3002;
+const port = 3002;
+const express = require('express');
+// const path = require('path');
```

Comment:
Commented code should reviewed and deleted if not needed

Reply:
Acknowledged

Reply:
Acknowledged and corrected.